docs/changelog.rst

```diff
@@ -3,11 +3,64 @@
 Changelog
 =========
 
+5.1.0
+-----
+
+:Released: 06.03.2025
+:Full Changelog: `v5.0.0...v5.1.0 <https://github.com/useblocks/sphinx-needs/compare/5.0.0...2aa4c0dfab5aa8f71b52126206d7f05c06ab09d3>`__
+
+The :ref:`needs_global_options` configuration option has been updated to a new format,
+to be more explicit and to allow for future improvements :pr:`1413`.
+The old format is currently still supported, but will emit a warning.
+Additionally, checks are put in place to ensure that the keys used are from the allowed set (:pr:`1410`).:
+
+- any ``needs_extra_options`` field
+- any ``needs_extra_links`` field
+- ``status``
+- ``layout``
+- ``style``
+- ``tags``
+- ``constraints``
+
+.. code-block:: python
+    :caption: Old format
```

Comment:
```suggestion
   :caption: Old format
```

Reply:
yehhh, but then it doesn't align with the python code in it